plugins/org.jboss.tools.openshift.client/.classpath

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
-	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry exported="true" kind="lib" path="lib/openshift-restclient-java-3.0.0-SNAPSHOT.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/commons-compress-1.8.1.jar"/>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry exported="true" kind="lib" path="lib/openshift-restclient-java-3.0.0-SNAPSHOT.jar" sourcepath="/openshift-restclient-java"/>
```

Comment:
@adietish I like this reference being their permanently, but did you intend to check it in?